### DISCUSSION TITLE
Does support HE mean support for tunnelbroker.net?

### DISCUSSION THREAD

**Discussion by tcurdt** [qdm12/ddns-updater#524] [Q&A] [answered]
Does support for "HE" mean it should be able to update the tunnelbroker.net configuration from HE?

Because I am not quite sure I can make sense of the docs for the configuration options.

- "domain"
- "host" is your host and can be a subdomain or "@" or "*" (untested)
- "password"

<img width="1112" alt="Screenshot 2023-09-04 at 22 36 44" src="https://github.com/qdm12/ddns-updater/assets/13697/78ce75a3-1283-4bf8-9d0d-f49629328ded">

**Comment by qdm12** [comment, marked as answer]
It only supports updates sent to `https://dyn.dns.he.net/nic/update`, I don't think it would work with tunnelbroker. Feel free to create an issue to support tunnelbroker though.
